server/oauth2.go

```diff
@@ -100,6 +100,7 @@ const (
 	errUnsupportedGrantType    = "unsupported_grant_type"
 	errInvalidGrant            = "invalid_grant"
 	errInvalidClient           = "invalid_client"
+	errInvalidConnectorID      = "invalide_connector_id"
```

Comment:
typo
```suggestion
	errInvalidConnectorID      = "invalid_connector_id"
```
Also, the others aren't made up, but part of the OAuth2 spec, see for example this section [5.2 Error Responses](https://tools.ietf.org/html/rfc6749#section-5.2). But, custom ones are allowed ([8.5.  Defining Additional Error Codes](https://tools.ietf.org/html/rfc6749#section-8.5)), so this is fine.